{"errorMessage": "Message: Unable to obtain driver for chrome; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors/driver_location\n", "errorType": "NoSuchDriverException", "requestId": "54f2865c-77ed-4e7a-9071-87c71b8dc15e", "stackTrace": ["  File \"/var/task/scraper_lambda.py\", line 13, in lambda_handler\n    futures, calls, puts = scrape_all_dates_full()\n", "  File \"/var/task/scraper/scrape_all_dates_full.py\", line 21, in scrape_all_dates_full\n    driver = setup_driver()\n", "  File \"/var/task/scraper/meff_scraper.py\", line 25, in setup_driver\n    driver = webdriver.Chrome(options=options)\n", "  File \"/var/task/selenium/webdriver/chrome/webdriver.py\", line 45, in __init__\n    super().__init__(\n", "  File \"/var/task/selenium/webdriver/chromium/webdriver.py\", line 51, in __init__\n    if finder.get_browser_path():\n", "  File \"/var/task/selenium/webdriver/common/driver_finder.py\", line 47, in get_browser_path\n    return self._binary_paths()[\"browser_path\"]\n", "  File \"/var/task/selenium/webdriver/common/driver_finder.py\", line 78, in _binary_paths\n    raise NoSuchDriverException(msg) from err\n"]}